beacon-chain/p2p/types/BUILD.bazel

```diff
@@ -9,8 +9,13 @@ go_library(
         "types.go",
     ],
     importpath = "github.com/prysmaticlabs/prysm/beacon-chain/p2p/types",
```

Comment:
This should probably be refactored into a better package. It can be done in another PR if you like, but these aren't specific to p2p.

Reply:
Will do in another PR.